src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="GET" 
+    data-qa="big" 
+    class="form"
+    >
+      <input 
+      class="searchInput firstSearch" 
+      type="search" 
+      placeholder="Try &quot;Los Angeles&quot;" 
+      data-qa="keypress"
+      >
+      <img src="images/Search.svg" 
+      alt="searchIcon" 
+      class="firstSvg icon">
```

Comment:
![Screenshot 2023-01-26 at 13 32 29](https://user-images.githubusercontent.com/84275496/214825507-d35c94b3-e0bf-4ca5-9093-510ddf4d9bf7.png)

Reply:
This issue is not fixed yet, check this checklist point to see good examples of code style